x-pack/solutions/security/plugins/security_solution/public/management/pages/policy/view/policy_settings_form/components/setting_card.tsx

```diff
@@ -73,16 +73,16 @@ export const SettingCard: FC<SettingCardProps> = memo(
     selected = true,
     mode = 'edit',
   }) => {
-    const paddingSize = useContext(ThemeContext).eui.euiPanelPaddingModifiers.paddingMedium;
     const getTestId = useTestIdGenerator(dataTestSubj);
+    const { euiTheme } = useEuiTheme();
 
     return (
       <EuiPanel data-test-subj={getTestId()} hasBorder={true} hasShadow={false} paddingSize="none">
         <EuiFlexGroup
           direction="row"
           gutterSize="none"
           alignItems="center"
-          style={{ padding: `${paddingSize} ${paddingSize} 0 ${paddingSize}` }}
+          style={{ padding: `${euiTheme.size.base} ${euiTheme.size.base} 0 ${euiTheme.size.base}` }}
```

Comment:
Is the inline style required? If not, I'd ask to use the `css` prop instead to add custom styles.

```ts
// using the euiTheme like this on the css prop, requires the appropriate emotion type setup
// in case you're plugin/package doesn't have it set up yet, see this related comment on required steps:
// https://github.com/elastic/kibana/pull/204537#discussion_r1888994894
css={({ euiTheme }) => ({
  padding: `${euiTheme.size.base} ${euiTheme.size.base} 0 ${euiTheme.size.base}`
})}
```

Same can be changed for the inline style usage further down.